### DISCUSSION TITLE
How to run standalone and allow for editing?

### DISCUSSION THREAD

**Discussion by PoweredByPeople** [reloadware/reloadium#79] [Q&A] [answered]
I see you mention you can use reloadium standalone, however I'm still a python beginner and can't figure out how to do it. 

When I run `reloadium run my_python_program.py` it just executes instantly and returns the results of my program. Is there a trick to get it to run alongside the vscode debugger (or potentially I need to use python built in debugger to pause code execution?) or would it be possible to share a simple example using a very basic python script in the docs?

**Comment by dkrystki** [comment, marked as answer]
Hi @PoweredByPeople 

Your process has to be long lived or paused by debugger to use hot reloading.

Use below example to play around in run mode.

```
from time import sleep

a = 1  # Change this

def fun():
    print(f"a = {a}")
    sleep(1)


while True:
    fun()
```

VsCode is not yet fully supported at the moment.
